### DISCUSSION TITLE
Open in New Incognito Tab?

### DISCUSSION THREAD

**Discussion by aidken** [fastaddons/GroupSpeedDial#315] [Q&A] [answered]
When you right-click a dial, there is "Open in new tab" option. Would you add another option "Open in new incognito tab"?

**Comment by Juraj-Masiar** [comment, marked as answer]
Hello,
It's already there - see Options page / Context menu / Private window:
![image](https://github.com/fastaddons/GroupSpeedDial/assets/6358971/1453cfad-4a55-44b8-87f7-1d70e38280c2)

Note that this option will work only if you enable my extension to run in Incognito mode.
You can enable it in Chrome page chrome://extensions/ or in Firefox addons page: about:addons

**Reply by aidken** [reply]
Ahh, thanks for quick reply. Sorry, I should have searched.

Regards,

**Reply by Juraj-Masiar** [reply]
No problem :),
Also, you are not the first, so maybe it's not intuitive enough.

**Reply by aidken** [reply]
Right... Open up web browser, right-click a link and I typically see "Open in Private/Incognito Tab" option. For me at least, "Open in Private/Incognito Tab" is something I expect to see. Maybe it's okay to show it as default.

Thanks anyways!
